src/components/ContentPreview/PreviewLoading.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow
+ * @file Editable transcript row component
+ * @author Box
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import IconFileDefault from 'box-react-ui/lib/icons/file/IconFileDefault';
+import LoadingIndicator from 'box-react-ui/lib/components/loading-indicator/LoadingIndicator';
+import messages from '../messages';
+import './PreviewLoading.scss';
+
+type Props = {
+    isError: boolean
+};
+
+const PreviewLoading = ({ isError }: Props) => (
+    <div className='bcp-loading-wrapper'>
+        {!isError ? (
```

Comment:
Take a look at box-react-ui/src/components/loading-indicator/makeLoadable.js

also, normally try to have positive conditions 1st, so `isError ? :` instead of `!isError ? :`